src/test/java/org/opensearch/security/authtoken/jwt/JwtVendorTest.java

```diff
@@ -152,7 +172,9 @@ public void testCreateJwtWithBadEncryptionKey() {
         List<String> roles = List.of("admin");
         Integer expirySeconds = 300;
 
-        Settings settings = Settings.builder().put("signing_key", "abc123").build();
+        Settings settings = Settings.builder()
+            .put("signing_key", "abc123-1234567812345678123456781234567812345678123456781234567812345678")
```

Comment:
Hi @MaciejMierzwa, Is there any key format restriction for this library? What is the story behind this test "signing_key" change?

Reply:
Yes, it turns out that by default library forces us to use keys of certain length in this case it's 256 bit's. Otherwise tests throw:

```
OpenSearchException[com.nimbusds.jose.KeyLengthException: The secret length must be at least 256 bits
]; nested: KeyLengthException[The secret length must be at least 256 bits];
	at org.opensearch.security.authtoken.jwt.JwtVendor.createJwkFromSettings(JwtVendor.java:83)
	at org.opensearch.security.authtoken.jwt.JwtVendorTest.testCreateJwkFromSettings(JwtVendorTest.java:42)
...
Caused by: com.nimbusds.jose.KeyLengthException: The secret length must be at least 256 bits
	at com.nimbusds.jose.crypto.impl.MACProvider.<init>(MACProvider.java:118)
	at com.nimbusds.jose.crypto.MACSigner.<init>(MACSigner.java:130)
	at com.nimbusds.jose.crypto.MACSigner.<init>(MACSigner.java:178)
	at org.opensearch.security.authtoken.jwt.JwtVendor.createJwkFromSettings(JwtVendor.java:81)
	... 28 more
```

Currently looking for a solution, either padding the keys or some switch that would allow shorter ones